sys/include/net/gnrc/gomach/gomach.h

```diff
@@ -0,0 +1,337 @@
+/*
+ * Copyright (C) 2016 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_gomach A traffic-adaptive multi-channel MAC
+ * @ingroup     net_gnrc
+ * @brief       A traffic adaptive MAC protocol that provides high traffic adaptability,
+ *              high energy efficiency and high robustness.
+ * @{
+ *
+ * @file
+ * @brief       Implementation of GoMacH protocol
+ *
+ * @author      Shuguo Zhuo <shuguo.zhuo@inria.fr>
+ */
+
+#ifndef NET_GNRC_GOMACH_GOMACH_H
+#define NET_GNRC_GOMACH_GOMACH_H
+
+#include "periph/rtt.h"
+#include "kernel_types.h"
+#include "net/gnrc/netif.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief The default duration of GoMacH's wake-up period (WP).
+ *
+ * GoMacH adopts the duty-cycle scheme that, by default, a node only wakes up for a short
+ * period of @ref GNRC_GOMACH_CP_DURATION_US in each cycle. In the rest of the cycle (except vTDMA),
+ * the node turns off the radio to conserve power. @ref GNRC_GOMACH_CP_DURATION_US should be at
+ * least longer than @ref GNRC_GOMACH_MAX_PREAM_INTERVAL_US, thus to guarantee that the
+ * receiver will not miss the preamble packet.
+ */
+#ifndef GNRC_GOMACH_CP_DURATION_US
+#define GNRC_GOMACH_CP_DURATION_US        (10U *US_PER_MS)
```

Comment:
Here and in other places: Please use one space on *each side* of an operator: `10 * US_PER_MS`.

Reply:
Em, I also found this abnormal, but uncrutify gave me this results. Anyway, will update.

Reply:
I think it thought those were pointers.